packages/eslint-config-typescript/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Would you please sort it using `npx sort-package-json`?

Reply:
sure